wme_street_to_river_plus.user.js

```diff
@@ -50,6 +50,7 @@ console.warn('Remove this line, when WME-Bootstrap will fix its syntax. now it c
     const idStreetToOther = 8;
     const idStreetToForest = 9;
     const idDeleteSegment = 10;
+	const idStreetToCanal = 11;
```

Comment:
here and below - please, use the same indentation symbols (tabs or spaces) as the old code has.